### PR TITLE
About page and Privacy Policy updates

### PR DESCRIPTION
**Pull request by maxious**
### What this PR does

Fixes #<insert issue number here if relevant>

### Checklist

-   [ ] There are unit tests to verify my changes are correct or unit tests aren't applicable
-   [ ] I've updated CHANGES.md with what I changed.
-   [ ] I've linked this PR to an issue in ZenHub (core dev team only)
-   [ ] I've assigned this PR to someone (if you don't know who to assign it to, pick @AlexGilleran)

### CODE REVIEW

File: magda-web-client/src/markdown/about.md [hunk 1]
```diff
@@ -2,32 +2,121 @@
...
+
+Information will only be used for purposes for which your provide it. Please see our Privacy policy for further information.
+
+### How do I register or upload data
+
+Anyone can [register an account](https://data.gov.au/user/register) on data.gov.au which can be used for commenting or liking datasets.
+
+But to publish government data you must have an account with an Australian Government email address (ending with “.gov.au”).
+
+Before you consider registering an account for a government (federal, state, local) or government-funded research agency, you should do a quick search of existing[organisations](https://search.data.gov.au/organisations) to see if your own agency is already listed. The listed contact can help you to log in and publish your data sets. If not, you can register for the first time.
```
Comment (AlexGilleran) [inline magda-web-client/src/markdown/about.md]: Missing a space between existing and organisations

File: magda-web-client/src/markdown/about.md [hunk 2]
```diff
@@ -2,32 +2,121 @@
...
+
+### How to attribute datasets
+
+When using content from this website that is licensed under a Creative Commons Licence, you are required to attribute the work in the manner specified in the licence (but not in any way that suggests that the publishing organisation endorses you or your use of the work).
+
+This website requires that you use the following form of attribution:
+
+Attribution to: Organisation name, jurisdiction, title of dataset, date the content was sourced, dataset URL
+
+Example:The Department of the Primary Industries and Regions, South Australia, Field Crop Estimates, Sourced on 22 July 2013, [http://www.data.sa.gov.au/dataset/field-crop-production-estimates](http://www.data.sa.gov.au/dataset/field-crop-production-estimates)
```
Comment (AlexGilleran) [inline magda-web-client/src/markdown/about.md]: Missing a space after Example:

File: magda-web-client/src/markdown/privacy-policy.md [hunk 3]
```diff
@@ -1,59 +1,69 @@
...
+data.gov.au is an online service managed by the Digital Transformation Agency. We respect and protect the privacy of our users. This privacy policy tells you how we collect and use information.
 
----
+The term "Personal Information" in this privacy policy means any information from which your identity is apparent or can be reasonably ascertained. We do not collect Personal Information about you when you visit this online service.
+
+You can use this service without telling us who you are or revealing other Personal Information. If you send us a contact or feedback form you do not need to identify yourself or use your real name.
+
+### Location of Content
+
+Content on the data.gov.au main site, search index, administrator account information and completed form information is hosted on Amazon servers located within Australia.
```
Comment (AlexGilleran) [inline magda-web-client/src/markdown/privacy-policy.md]: Not sure if you want to change this to Google?

**Comment by maxious** [comment]
Fixed spaces and added both clouds to be flexible.

**Pull request merged**